packet/utils.py

```diff
@@ -70,6 +72,25 @@ def wrapped_function(*args, **kwargs):
     return wrapped_function
 
 
+def admin_auth(func):
+    """
+    Decorator for easily configuring oidc
+    """
+    @auth.oidc_auth('app')
+    @wraps(func)
+    def wrapped_function(*args, **kwargs):
+        if app.config['REALM'] == 'csh':
```

Comment:
This would mean if for some reason this gets used on a frosh route, the route would be accessible, wouldn't it?